mappings/net/minecraft/entity/attribute/EntityAttributeModifier.mapping

```diff
@@ -28,7 +28,16 @@ CLASS net/minecraft/class_1322 net/minecraft/entity/attribute/EntityAttributeMod
 	METHOD method_6186 getValue ()D
 	METHOD method_6189 getId ()Ljava/util/UUID;
 	CLASS class_1323 Operation
+		COMMENT Represents an operation which can be applied to an attribute modifier.
+		FIELD field_6328 ADDITION Lnet/minecraft/class_1322$class_1323;
+			COMMENT Adds to the base value of an attribute.
 		FIELD field_6329 id I
+		FIELD field_6330 MULTIPLY_BASE Lnet/minecraft/class_1322$class_1323;
+			COMMENT Multiplies the base value of the attribute.
```

Comment:
does multiply base go before addition? if yes i'd prefer to specify that

Reply:
Multiply base occurs after addition, but uses the base value.